### PR TITLE
PROD-545: Update home stats box

### PR DESCRIPTION
**Pull request by neel-ds**
- Description
Add more stats box and rearrange the grid

- What are the steps to test that this code is working?
You can see the new stats box on /application

- Screen shots or recordings for UI changes
Screenshot 2024-12-11 at 7 13 47 PM

### CODE REVIEW

**Comment by marvinmarnold** [comment]
Conflicts now fyi

**Comment by neel-ds** [comment]
> Conflicts now fyi

It's resolved now. Waiting for modal description.

File: app/components/StatsBox/StatsBox.tsx [hunk 1]
```diff
@@ -24,9 +25,23 @@ export function StatsBox({
...
+      case "claimed":
+        return "text-chomp-orange-light";
+      case "points":
+        return "text-chomp-green-light";
+      case "credits":
+        return "text-chomp-blue-light";
+      default:
+        return "text-white";
+    }
+  };
```
Comment (samott) [inline app/components/StatsBox/StatsBox.tsx:43]: How about using a custom type for this prop instead of a string?

Comment (samott) [reply app/components/StatsBox/StatsBox.tsx:41]: yep, looks good. Thanks!

**Review by samott — APPROVED** [review]

**Comment by marvinmarnold** [comment]
Looks good. Just waiting on all the copy.

**Pull request merged**